### ISSUE TITLE
NPM package is unusable

### ISSUE THREAD

**Issue by binyamin** [arnelenero/simpler-color#1]
The NPM package, in both `/lib` and `/es`, references `@babel/runtime`, which is a devDependency. I tried importing the package with traditional node JavaScript, ESM, and typescript. They all failed.

Solution? Run the typescript compiler first, and just export the emitted JavaScript. You can work on ESM compatibility later, but nobody can use it now.

**Comment by binyamin** [comment]
I ran the source files through [esbuild](esbuild.github.io/), and it worked fine.
```sh
esbuild --bundle ./src/index.ts --platform=node --format=esm --outfile=simpler-color.js
```

**Comment by arnelenero** [comment]
Hi @binyamin .  Thanks for bringing this to my attention.  I missed to include `@babel/runtime` as runtime dependency. Funny thing is that it worked on my test apps because those React apps indirectly import @babel/runtime via one of their direct runtime dependencies.

I published a new version (v0.1.3) and tested using regular Node javascript with no other dependencies but simpler-color. It appears to be working now. Please let me know if there are still issues.

I really appreciate this issue report. Can't thank you enough.

**Comment by binyamin** [comment]
It works perfectly, thank you.
